### PR TITLE
Include toots/polls.md

### PR DESCRIPTION
**Pull request by thilobuchholz**
This pull requests adds a section on how to create and participate in polls on Mastodon.
\+ 4 images in toots/img/ which are used in polls.md
\+ minor typo fixes in toots/visibility.md
\- "polls" in the todo-list in docs/readme.md

### CODE REVIEW

File: docs/toots/polls.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+### Auslaufdauer
+Rechts unten kannst du die Auslaufdauer der Umfrage einstellen. Nach der Zeitspanne, die du einstellst, wird die Umfrage dann automatisch geschlossen. Du hast die Wahl zwischen Zeiträumen von 5 Minuten bis zu 7 Tagen (standardmäßig ist 1 Tag ausgewählt):
+
+![5 Minuten, 30 Minuten, 1 Stunde, 6 Stunden, 1 Tag, 3 Tage, 7 Tage](img/toot-polls-expiration-date.png)
+
+## An Umfragen teilnehmen
+
+### Eine Umfrage ausfüllen
+Wann immer jemand eine Umfrage erstellt hat, die noch nicht abgelaufen ist, und die für dich ![sichtbar ist](visibility.md), kannst du an dieser Umfrage teilnehmen. Du hast je nach den Einstellungen, die die erstellende Person gewählt hat, die Wahl zwischen 2-4 Möglichkeiten, und kannst entweder eine oder mehrere Stimmen abgeben. Außerdem kannst du sehen, wie viele Nutzer\*innen bereits an der Abstimmung teilgenommen haben, und wie lange die Umfrage noch offen sein wird. Wie auch bei anderen Toots kannst du auf den Toot antworten, oder die Umfrage favorisieren oder boosten.
```
Comment (marians) [inline docs/toots/polls.md]:
```suggestion
Wann immer jemand eine Umfrage erstellt hat, die noch nicht abgelaufen ist, und die für dich [sichtbar ist](visibility.md), kannst du an dieser Umfrage teilnehmen. Du hast je nach den Einstellungen, die die erstellende Person gewählt hat, die Wahl zwischen 2-4 Möglichkeiten, und kannst entweder eine oder mehrere Stimmen abgeben. Außerdem kannst du sehen, wie viele Nutzer\*innen bereits an der Abstimmung teilgenommen haben, und wie lange die Umfrage noch offen sein wird. Wie auch bei anderen Toots kannst du auf den Toot antworten, oder die Umfrage favorisieren oder boosten.
```

Das Ausrufezeichen am Anfang wird nur bei Bildern verwendet.

**Comment by marians** [comment]
Fantastic, thanks!

Two changes:

- One syntax change for the hyperlink
- Please also edit `docs/_sidebar.md` to contain the new page.

Also thanks for the boyscouting!

**Comment by thilobuchholz** [comment]
>     * Please also edit `docs/_sidebar.md` to contain the new page.

Done! I have to admit, I was distracted by docs/toots/_sidebar.md
Is there a reason for this file to exist? Or can we delete it?

**Comment by marians** [comment]
> Or can we delete it?

I think it has to exist in order for nested navigation to work.

**Comment by marians** [comment]
Can you please move "Umfragen" below "Hashtags"? I think they are a little bit more of a special topic while Hashtags concern pretty much everyone, frequently.

**Pull request merged**